website/content/docs/concepts/domain-model/credential-libraries.mdx

```diff
@@ -7,39 +7,70 @@ description: |-
 
 # Credential Libraries
 
-A credential library is a resource
-that provides [credentials][]
-of the same type and same access level
-from a single [credential store][].
+A credential library is a resource that provides [credentials][] of the same type and same access level from a single [credential store][].
 
 ## Attributes
 
-A credential library has the following configurable attributes:
+A credential library has the following optional configurable attributes:
 
-- `name` - (optional)
-  If set, the `name` must be unique within the credential library's parent
-  [credential store][].
+- `name` - If you set this attribute, the `name` must be unique within the credential library's parent [credential store][].
 
-- `description` - (optional)
+- `description` - A user-defined description of the credential library for identification purposes.
 
-### Vault Credential Library Attributes
+### Vault generic credential library attributes
 
-A Vault credential library has the following additional attributes:
+The generic Vault credential library has the following additional attributes:
 
-- `path` - (required)
-  The path in Vault to request credentials from.
+- `path` - (required) The path in Vault to request credentials from.
 
-- `http_method` - (optional: defaults to `GET`)
-  The HTTP method the library uses when requesting credentials from Vault.
-  Can be either `GET` or `POST`.
+- `http_method` - The HTTP method the library uses when requesting credentials from Vault.
+Can be either `GET` or `POST`.
+The default value is `GET`.
 
-- `http_request_body` - (optional)
-  The body of the HTTP request the library sends to Vault when requesting credentials.
-  Only valid if `http_method` is set to `POST`.
+- `http_request_body` - The body of the HTTP request the library sends to Vault when requesting credentials.
+Only valid if `http_method` is set to `POST`.
 
-### Vault Credential Library Parameter Templating
+### Vault SSH certificate credential library attributes
 
-In some cases it can be useful to provide information about a Boundary user or account when making a call to Vault. For example, this can allow picking the correct role when asking for database credentials (if roles are separated per-user), or providing a value to encode in an X.509 certificate generated by Vault. As of Boundary 0.11.1, you can template user and account information into either the path in Vault, the `POST` request body, or both.
+As of Boundary 0.12.0, you can configure SSH credential injection using Vault.
```

Comment:
I think this need to be rephrased slightly. Earlier versions support SSH credential injection using vault, but only via the generic credential library and only using `username_password` or `ssh_private_key` credential types.

The new thing in 0.12.0 is a new credential library that enables SSH credential injection using Vault's SSH secret engine to create SSH Certificate credentials.

Reply:
Good catch! Thanks.